framework/CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ Yii Framework 2 Change Log
 2.0.40 under development
 ------------------------
 
-- no changes in this release.
+- Enh #18381: The `yii\web\AssetManager` `$basePath` writeable check has been moved to the initializer of the class instead of making this check whenever a file will be pbulished (nadar)
```

Comment:
```suggestion
- Enh #18381: The `yii\web\AssetManager` `$basePath` writeable check has been moved to the initializer of the class instead of making this check whenever a file will be published (nadar)
```